src/observer/ingest_logic.py

```diff
@@ -356,7 +359,67 @@ def regenerate_all():
     regenerate_many(logic.known_articles())
 
 #
-# upsert from file/dir of article-json
+# upsert article-json from api
+#
+
+if settings.ENV not in ['prod', 'continuumtest']:
```

Comment:
I can do that. Looking at it now I think hard coding env names is a smell in general. Will switch it to `if settings.DEBUG`